<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Fahrenheit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="outFah">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 191, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inFahr">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(116, 116, 116);</string>
         </property>
         <property name="maximum">
          <number>210</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Cent√≠grados</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLCDNumber" name="outCent">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 165, 240);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDial" name="inCent">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(143, 143, 143);</string>
         </property>
         <property name="minimum">
          <number>-18</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(172, 172, 172);</string>
      </property>
      <property name="title">
       <string>Kelvin</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLCDNumber" name="outKel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(139, 255, 172);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inKel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outFah</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inKel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outKel</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>738</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>738</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
